x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/patch_rules_schema.test.ts

```diff
@@ -1063,6 +1063,148 @@ describe('patch rules schema', () => {
     });
   });
 
+  test('You cannot send in an array of actions that are missing "group"', () => {
+    expect(
+      patchRulesSchema.validate<
+        Partial<
+          Omit<PatchRuleAlertParamsRest, 'actions'> & {
+            actions: Array<Omit<AlertAction, 'group'>>;
+          }
+        >
+      >({
+        actions: [{ id: 'id', actionTypeId: 'actionTypeId', params: {} }],
+        rule_id: 'rule-1',
+        risk_score: 50,
+        description: 'some description',
+        name: 'some-name',
+        severity: 'junk',
```

Comment:
To avoid a possible false positive or weirdness, I would set the severity here to be something valid such as: `severity: 10`, This is still a good test as it is testing what you want it to be but it should be set to a real number so everyone can tell it is valid other than what you are testing.

I kind of don't want to make a comment here because you have such clean beautiful tests exercising everything but this would make your already clean nice tests even better.

Reply:
Fixed, Thank you @FrankHassanabad 👍